src/openrct2-ui/input/ShortcutManager.h

```diff
@@ -78,6 +79,14 @@ namespace OpenRCT2::Ui
         {
         }
 
+        RegisteredShortcut(std::string_view id, StringId localisedName, const std::function<void()>& action, int16_t index)
```

Comment:
```suggestion
        RegisteredShortcut(std::string_view id, StringId localisedName, const std::function<void()>& action, size_t index)
```

Can probably be combined with the previous constructor using a default value. Same goes for the other added constructors.